pkg/api/versions.go

```diff
@@ -109,21 +106,21 @@ func (instance *Instance) IsVersioned(data []byte) ([]*Output, error) {
 }
 
 // containsStub checks to see if a []byte has a stub in it
-func containsStub(data []byte) ([]*Stub, error) {
+func containsStub(data []byte) []*Stub {
 	klog.V(10).Infof("\n%s", string(data))
 	stub, err := jsonToStub(data)
 	if err != nil {
 		klog.V(8).Infof("invalid json: %s", err.Error())
 	} else {
-		return stub, nil
+		return stub
 	}
 	stub, err = yamlToStub(data)
 	if err != nil {
 		klog.V(8).Infof("invalid yaml: %s", err.Error())
 	} else {
-		return stub, nil
+		return stub
 	}
-	return nil, err
```

Comment:
Isn't this still a valid error return. If both json and yaml error shouldn't it fail? This error bubbles up all the way to each individual detect type.

Reply:
I suppose technically it is.  However, when we call it, we just need to know if there's anything in the stub (which we check with len() ).  We log any errors that we get, so I'm not sure what else we would do with it.

Reply:
Eh, I suppose one of the underlying functions could throw a real error. I will put it back

Reply:
I put that handling back and added a test case that covers it as well